### ISSUE TITLE
Terraform Pipeline Broken

### ISSUE THREAD

**Issue by shellac85** [hashicorp/terraform-provider-aws#16666]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform AWS Provider Version
provider.aws: version = "~> 3.20"
I am not sure where to find my Terraform Version

### Affected Resource(s)
aws_lambda_function

* aws_XXXXX

### Terraform Configuration Files

Please include all Terraform configurations required to reproduce the bug. Bug reports without a functional reproduction may be closed without investigation.

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp
```

### Expected Behavior
The Jenkins Pipeline should not fail

### Actual Behavior
The Jenkins build is failing with the error:?
```
error getting Lambda Function (watchtower_backend_testing) code signing config AccessDeniedException: 
	status code: 403, request id: f3fcd2bb-f8a2-4bf1-a712-d24fc69c9ac5
```

### Steps to Reproduce

1. `terraform apply`

### Important Factoids
This just happened yesterday and I have no way to update other AWS items like RDS engine version. Any help greatly appreciated.

### References

* #0000

**Comment by dthvt** [comment]
This is an authorization issue w/ the credentials Jenkins is using to connect to the AWS API.  Check the permissions assigned to the Jenkins user/role in IAM.

**Comment by shellac85** [comment]
Not sure why you think this is an authorisation issue, I eventually fixed this by manually putting version 3.15 in out Terraform code. Terraform version 3.20 broke this.

**Comment by dthvt** [comment]
Then there must be a code change between 3.15 - 3.20 that started using a new API on the AWS side.  Regardless, the error you encountered is an authorization error and can be resolved by updating the permissions assigned to your Jenkins build.
